clouderror/wrap.go

```diff
@@ -33,11 +33,11 @@ func WrapTransient(err error, msg string) error {
 // WrapTransientCaller wraps transient errors with an appropriate codes.Code and caller.
 // The returned error will always be a status.Status with description set to msg.
 func WrapTransientCaller(err error, msg string, caller Caller) error {
+	if err == nil {
+		return &wrappedStatusError{status: status.New(codes.Internal, msg), err: nil, caller: caller}
```

Comment:
Shouldn't this be something like

```suggestion
		return &wrappedStatusError{status: status.New(codes.OK, msg), err: nil, caller: caller}
```

Do we have the history on why we mapped it to internal to begin with?

Reply:
Added that to keep the tests intact, but open for discussion on how to handle this case

Reply:
[This](https://github.com/einride/cloudrunner-go/blob/master/clouderror/wrap_test.go#L23) is the case I'm referring to